src/Zicht/Bundle/HtmldevBundle/Twig/ValidateExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @copyright Zicht Online <http://zicht.nl>
+ */
+namespace Zicht\Bundle\HtmldevBundle\Twig;
+
+use Rakit\Validation\Validator;
+use Twig_Extension;
+use Twig_SimpleFilter;
+
+/**
+ * Twig extensions for validating data.
+ */
+class ValidateExtension extends Twig_Extension
```

Comment:
`extends \Twig\Extension\AbstractExtension`?

Reply:
Hmm... Er is geen Twig versie gespecificeerd in deze bundle, dus :point_up_2: dat is geen goed idee :wink: 

Zie #41

Reply:
Dit is een gevalletje copy-paste van de andere extentions